pyravendb/tools/utils.py

```diff
@@ -291,7 +292,11 @@ def timedelta_to_str(timedelta_obj):
                 timedelta_str += "{0}.".format(days)
             timedelta_str += "{:02}:{:02}:{:02}".format(hours, minutes, seconds)
             if microseconds > 0:
-                timedelta_str += ".{0}".format(microseconds)
+                timedelta_str += "."
+                order_of_magnitude = log10(microseconds).__trunc__()
+                if order_of_magnitude <= 5:
+                    timedelta_str += "0" * (5 - order_of_magnitude)
```

Comment:
isnt that just padding? https://www.w3schools.com/python/trypython.asp?filename=demo_ref_string_ljust2

Reply:
done